pages/matches/[id].js

```diff
@@ -24,10 +51,12 @@ const currentBars = currentMatch ? (bars.filter((bar) => (bar.matches.includes(c
         {currentMatch ? <SelectedMatch date={currentMatch.date} time={currentMatch.time} homeTeam={currentMatch.homeTeam.name} homeTeamLogoColor={currentMatch.homeTeam.logoColor} awayTeam={currentMatch.awayTeam.name} awayTeamLogoColor={currentMatch.awayTeam.logoColor}/> : <h2>loading</h2>}
         <StyledHeadline>Folgende Bars zeigen das Spiel:</StyledHeadline>
```

Comment:
Just for consistency: I would recommend that if you want to use the word "Styled" for the naming of your styled components, you should add "Styled" to every styled component. Otherwise maybe remove it for this one and maybe for all other styled components where you have added "Styled"? 🤔

Reply:
Done :-)